ergotree-ir/src/chain/preheader.rs

```diff
@@ -26,6 +22,20 @@ pub struct PreHeader {
     pub votes: Votes,
 }
 
+impl From<Header> for PreHeader {
```

Comment:
Any chance `PreHeader` can be moved as well? 

EDIT: I don't see a reason for it to stay here.

Reply:
Done.